datalad/distribution/tests/test_create_sibling.py

```diff
@@ -78,44 +78,12 @@
 import logging
 lgr = logging.getLogger('datalad.tests')
 
-
-def assert_publish_with_ui(target_path, rootds=False, flat=True):
-
-    paths = [_path_(".git/hooks/post-update")]     # hooks enabled in all datasets
-    not_paths = []  # _path_(".git/datalad/metadata")]  # metadata only on publish
-                    # ATM we run post-update hook also upon create since it might
-                    # be a reconfiguration (TODO: I guess could be conditioned)
-
-    # web-interface html pushed to dataset root
-    web_paths = ['index.html', _path_(".git/datalad/web")]
-    if rootds:
-        paths += web_paths
-    # and not to subdatasets
-    elif not flat:
-        not_paths += web_paths
-
-    for path in paths:
-        ok_exists(opj(target_path, path))
-
-    for path in not_paths:
-        assert_false(exists(opj(target_path, path)))
-
-    hook_path = _path_(target_path, '.git/hooks/post-update')
-    # No longer the case -- we are no longer using absolute path in the
-    # script
-    # ok_file_has_content(hook_path,
-    #                     '.*\ndsdir="%s"\n.*' % target_path,
-    #                     re_=True,
-    #                     flags=re.DOTALL)
-    # No absolute path (so dataset could be moved) in the hook
-    with open(hook_path) as f:
-        assert_not_in(target_path, f.read())
-    # correct ls_json command in hook content (path wrapped in "quotes)
-    ok_file_has_content(hook_path,
-                        '.*datalad ls -a --json file \..*',
-                        re_=True,
-                        flags=re.DOTALL)
-
+try:
+    from datalad_deprecated.tests.test_sibling_webui \
```

Comment:
could there be at least a single matrix run (which would cover these tests) with `datalad_deprecated` installed?  or may be the entire test just copied over to deprecated and test here would not test for this aspect at all.  And then we add that dedicated github action for `datalad_deprecated` extension

Reply:
ATM they are very intertwined. I'd say it would make sense in the extension to simply import the test suite and run it in full, in a dedicated extension test action, as you suggested.

Reply:
Done